### PR TITLE
Fix support for node v0.8 support

### PR DESCRIPTION
**Pull request by kumavis**
@ljharb

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "browser-pack": "^6.0.1",
     "browser-resolve": "^1.11.0",
     "browserify-zlib": "~0.2.0",
-    "buffer": "^5.0.2",
+    "buffer": "5.2.1",
```
Comment (ljharb) [inline package.json]:
```suggestion
    "buffer": "~5.2.1",
```

File: package.json [hunk 2]
```diff
@@ -84,7 +84,7 @@
     "semver": "^5.5.0",
     "seq": "0.3.5",
     "tap": "^10.7.2",
-    "temp": "^0.8.1",
+    "temp": "0.8.3",
```
Comment (ljharb) [inline package.json]:
```suggestion
    "temp": "~0.8.3",
```
?

Comment (kumavis) [reply package.json]: the breakage here is in `0.8.4`

Comment (ljharb) [reply package.json]: edited to show the intent with = then :-)

**Comment by kumavis** [comment]
I also noticed appveyor and travis tests dont have the same engine coverage...

**Comment by kumavis** [comment]
would be nice to document supported platforms for both browserify and bundles

**Comment by kumavis** [comment]
also if we could turn on greenkeeper for only in-range dep updates? would give us an idea if our deps break

**Comment by kumavis** [comment]
woof appveyor takes forever

**Comment by kumavis** [comment]
@ljharb finished cooking

**Comment by ljharb** [comment]
We should definitely make sure “engines” is set correctly, as are CI matrices.

**Pull request merged**